lib/Sema/CSDiagnostics.cpp

```diff
@@ -2613,8 +2613,7 @@ bool ContextualFailure::tryProtocolConformanceFixIt(
   // Let's build a list of protocols that the context does not conform to.
   SmallVector<std::string, 8> missingProtoTypeStrings;
   for (auto protocol : layout.getProtocols()) {
-    if (getTypeChecker()
-            .conformsToProtocol(FromType, protocol->getDecl(), getDC(),
+    if (TypeChecker::conformsToProtocol(FromType, protocol->getDecl(), getDC(),
                                 ConformanceCheckFlags::InExpression)
```

Comment:
Nit: `ConformanceCheckFlags::InExpression` needs indenting

Reply:
Good catch. I'll re-indent this after I rebase onto your changes.